src/res/translation/translation_zh_CN.ts

```diff
@@ -0,0 +1,3072 @@
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="67"/>
+        <source>Server address edit box</source>
+        <translation>服务器地址编辑框</translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="68"/>
+        <source>Holds the current server IP address or URL. It also stores old URLs in the combo box list.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="73"/>
+        <source>Server List Selection</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="73"/>
+        <source>Selects the server list to be shown.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="74"/>
+        <source>Server list selection combo box</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="77"/>
+        <source>Filter</source>
+        <translation>过滤器</translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="78"/>
+        <source>The server list is filtered by the given text. Note that the filter is case insensitive.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="80"/>
+        <source>Filter edit box</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="83"/>
+        <source>Show All Musicians</source>
+        <translation type="unfinished">显示所有乐手</translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="84"/>
+        <source>If you check this check box, the musicians of all servers are shown. If you uncheck the check box, all list view items are collapsed.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="86"/>
+        <source>Show all musicians check box</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../connectdlg.cpp" line="136"/>
+        <source>Filter text, or # for occupied servers</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>CConnectDlgBase</name>
+    <message>
+        <location filename="../../connectdlgbase.ui" line="14"/>
+        <source>Connection Setup</source>
+        <translation>连接配置</translation>
+    </message>
+    <message>
+        <source>List</source>
+        <translation type="vanished">列表</translation>
+    </message>
```

Comment:
I think this (and all the other translations which have vanished) don't need a translation?

Reply:
Yes, I actually did the translation by editing the italian ts file and generate qm file use the linguist application and then I realized Jamulus did embed the qm files to resource file instead distribute qm files inside the translation folder. Then I cloned the full source, added the ts and qm file that I already translated, then run `lrelease` and reworked some translation and compile it manually. The vanished strings are the ones I did translated before I get compile the program by myself.